uint32 frameNum
string framePath
geometry_msgs/Point[] coordinates
geometry_msgs/Vector3[] speed